server/jobs/onboarding/freckleJob.js

```diff
@@ -0,0 +1,11 @@
+import { fetchNewPlacements } from '../../modules/allocations';
+import { createProject } from '../../modules/freckle/projects';
+
+export default () => {
+  fetchNewPlacements('Placed - Awaiting Onboarding')
+    .then((engagements) => {
+      engagements.map(engagement => (
+        createProject(engagement.client_name)
```

Comment:
The onboarding freckle automation is not just to create projects alone, it also involves assigning the devs placed to the project created.
You might want to take a look at the story again.